src/client/client-wrapper.ts

```diff
@@ -135,4 +135,30 @@ export class ClientWrapper {
     });
   }
 
+  /**
+   * Retrieves a single CampaignMember record for a given email address, including the
+   * provided field.
+   *
+   * @param {String} email - Email address of the CampaignMember record to retrieve.
+   * @param {String} field - CampaignMember field to include on the returned record.
+   */
+  // tslint:disable-next-line:max-line-length
+  public async findCampaignMemberByEmail(email: string, fields: string[]): Promise<Record<string, any>> {
```

Comment:
Thanks for the suggestion. I included campaignId as an arguement and also changed the name. As for findOne i assume it returns the first data it could find because when i checked there is also 'find' with the same arguements which i assume returns a list of data.